index.html

```diff
@@ -0,0 +1,344 @@
+</head>
+<header>
+	<div class="header__content">
+		<div class="header__img-wrapper">
+			<img class="header__img" src="./png/AX.png" alt="AX"> 
+			<img class="header__img" src="./png/IT.png" alt="IT">
+   		</div>
+    	<div class="header__menu">
+			<div>Features</div>
+			<div>About</div>
+			<div>Pricing</div>
+			<div>Reviews</div>
+			<div>Contact</div>
+		</div>
+	</div>
+</header>
+<body>
+		<main>
+		<section class="social">
+			<div class="social__content">
+			<div class="social__text-wrapper">
+				<div class="social__title">
+					<p class="social__title">Social Media</p>
+				</div>
+				<div class="social__subtitle">
+					<p class="social__subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+						Aenean euismod bibendum laoreet.</p>
+				</div>
+			</div>
+			<div class="social__icon-wrapper">
+				<div>
+					<i class="fa fa-facebook fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-twitter fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-linkedin-square fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-pinterest-square fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-instagram fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-github fa-4x" aria-hidden="true"></i>
+				</div>
+				<div>
+					<i class="fa fa-rss fa-4x" aria-hidden="true"></i>
+				</div>
+			</div>
+			</div>
+		</section>
+		<section class="tabs">
+			<div>
+				    <div class="tabs__button-wrapper">
+                        <button class="tabs__button-first" type="submit"> TAB 1 </button> <!--добавить рамку, тонкие буквы TAB-->
+                        <button class="tabs__button" type="submit"> TAB 2 </button>
+                        <button class="tabs__button" type="submit"> TAB 3 </button>
+                    </div>
+				    <div class="tabs__content">
+						    <div class="tabs__content-text">
+						    <div >
+				                <h3 class="tabs__content-title">Tabs with soft transitioning effect.</h3>
+			                </div>
+			                <div class="tabs__content-text-block">
+				            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod 
+					        bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra  
+					        justo commodo. Proin sodales pulvinar tempor. Cum sociis natoque 
+					        penatibus et magnis dis parturient montes.
+					       <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod 
+					        bibendum laoreet.</p>
+			                </div>
+							<div>
+								<button class="tabs__content-buttom">Download</button>
+							</div>
+						    </div>
+						    <div class="tabs__content-img">
+							    <img class="tabs__img" src="./png/NewYork.png" alt="Statue of Liberty in New York"> <!--чето не то с размером-->
```

Comment:
Something wrong with height is because you made 501px height for tabs__content element. Make it 500px and everything will be good.